src/components/Dashboard/TodoList.vue

```diff
@@ -130,9 +130,10 @@
                 </div>
 
                 <!-- pending filing - correction or alteration, or payment incomplete -->
-                <div v-else-if="isStatusPending(item)" class="todo-subtitle">
+                <div v-else-if="(isStatusPending(item) || isStatusPendingCorrection(item))" class="todo-subtitle">
                   <template v-if="isTypeCorrection(item) || isTypeAlteration(item)">
-                    <span>FILING PENDING</span>
+                    <span v-if="inProcessFiling === item.filingId">PROCESSING...</span>
+                    <span v-else>FILING PENDING</span>
```

Comment:
Like the other instances of filings where we check their status a few times a second apart, this one should say "processing..." while we check and "filing pending" after.